spec/features/current_organization_spec.rb

```diff
@@ -33,8 +33,13 @@
     let(:user) { create :user, :with_organizations }
 
     before :each do
-      within '#switch_organization' do
-        click_link user.organizations.last.name
+      click_on 'Change organization'
+      page.find("##{dom_id(user.organizations.last)}").click
```

Comment:
@yunixon why don't you use click on name? If you want to click on button in specified line then use:
```
within "##{dom_id(organization)}" do
  click_on 'Switch"
end
```